src/test/java/staffconnect/logic/commands/SortCommandTest.java

```diff
@@ -43,6 +44,7 @@ public void equals() {
         assertFalse(nameSortCommand.equals(new SortCommand(MODULE_COMPARATOR)));
         assertFalse(nameSortCommand.equals(new SortCommand(VENUE_COMPARATOR)));
         assertFalse(nameSortCommand.equals(new SortCommand(PHONE_COMPARATOR)));
+        assertFalse(nameSortCommand.equals(new SortCommand(FACULTY_COMPARATOR)));
```

Comment:
it would be better to arrage in alphabetic order

Reply:
I think better to follow person's order, for readability? Also following how edit and add did their ordering.